tendrl/commons/flows/expand_cluster/utils.py

```diff
@@ -0,0 +1,58 @@
+import copy
```

Comment:
This is slightly different, in the sense that during expand we need not write ssh-keys to authorized_key file of current node as current node is already a cluster node and this would have already happened.

If you feel that, the create_cluster utils has to be used, we will have to add a flag in create cluster util to achieve this. Please let me know which one is better.